### PR TITLE
Do not set minor OS version for Debian and Ubuntu

### PR DESCRIPTION
**Pull request by maximiliankolb**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.11/Katello 4.13
* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8)

### CODE REVIEW

**Comment by maximiliankolb** [comment]
@sbernhard Does this match your expectation? If so, please review/ACK.

**Review by sbernhard — APPROVED** [review]

File: guides/common/modules/proc_creating-operating-systems.adoc [hunk 1]
```diff
@@ -62,7 +68,12 @@ endif::[]
 --family "{client-os-family-hammer}" \
 --major {client-os-major} \
 --media "{client-os-family}" \
+ifndef::debian,ubuntu[]
 --minor {client-os-minor} \
+endif::[]
+ifdef::debian,ubuntu[]
+--release-name "{client-os-codename}" \
+endif::[]
```
Comment (Lennonka) [inline guides/common/modules/proc_creating-operating-systems.adoc:76]: Positive condition first. Please, swap `ifdef` and `ifndef`.

Comment (maximiliankolb) [reply guides/common/modules/proc_creating-operating-systems.adoc]: Applied.

**Review by Lennonka — APPROVED** [review]

**Comment by maximiliankolb** [comment]
Merged to "master" and cherry-picked:
   0e92600ef1..09e953fca7  3.11 -> 3.11
   9edd597de0..f2653b6fa8  3.10 -> 3.10
   b90f7121cd..5477c71902  3.9 -> 3.9

**Pull request merged**